server/models/Article.js

```diff
@@ -12,6 +12,12 @@ const schema = new Schema({
     visible: {
         type: Boolean,
         default: true,
+    },
+    language:{
+        type:String,
+        default:'En',
+        upsert: true,
```

Comment:
why upsert: true?

Reply:
I was trying to update the database to take default En but I think no need for upsert